lang/build.gradle

```diff
@@ -62,16 +62,70 @@ tasks.register('javadocJar', Jar) {
 
 publishing {
     publications {
-        create('maven', MavenPublication) {
-            artifactId = 'partiql-lang-kotlin'
+        maven(MavenPublication) {
+            // group id and version are set in the root build.gradle for all projects.
+            artifactId = "partiql-lang-kotlin"
+
+            from components.java
+            artifact sourcesJar
+            artifact javadocJar
 
-            from(components['java'])
-            artifact(tasks['sourcesJar'])
-            artifact(tasks['javadocJar'])
+            pom {
+                name = "PartiQL-Lang-Kotlin"
+                packaging = "jar"
+                url = "https://github.com/amzn/ion-schema-kotlin"
+                description = "An implementation of PartiQL for the JVM written in Kotlin."
+                scm {
+                    connection = "scm:git@github.com:partiql/partiql-lang-kotlin.git"
+                    developerConnection = "scm:git@github.com:partiql/partiql-lang-kotlin.git"
+                    url = "git@github.com:partiql/partiql-lang-kotlin.git"
+                }
+                licenses {
+                    license {
+                        name = "The Apache License, Version 2.0"
+                        url = "http://www.apache.org/licenses/LICENSE-2.0.txt"
+                    }
+                }
+                developers {
+                    developer {
+                        name = "Amazon Ion Team"
+                        email = "partiql-community@amazon.com"
```

Comment:
Is the e-mail correct? I don't know the correct one but wanted to double check since this one looks weird, imho it looks more like a discussion list e-mail

Reply:
I verified this with @jonwilsdon but I can put any email address here that the team recommends.  I wasn't sure our usual email list was appropriate, but perhaps it is.

Reply:
I'd suggest using ion-team@amazon.com here (coincides with "Amazon Ion Team").  If there's an appropriate way to reference https://community.partiql.org/, that might be a nice addition somewhere.

Reply:
sure--next commit.